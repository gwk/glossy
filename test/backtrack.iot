{
  'compile': ['$PROJ/legs/__main__.py test/$NAME.legs -test -output $NAME.swift'],
  'cmd': './$NAME.swift',
  'args': [
    'a aa b bb abc aabbcc',
    'ab aabb',
  ],
}