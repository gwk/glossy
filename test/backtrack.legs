s: \_+
a+
b+
a+b+c+
