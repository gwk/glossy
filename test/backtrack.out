
"a aa b bb abc aabbcc":
a_:0-1
s:1-2
a_:2-4
s:4-5
b_:5-6
s:6-7
b_:7-9
s:9-10
a_b_c_:10-13
s:13-14
a_b_c_:14-20

"ab aabb":
a_:0-1
s:1-3
a_:3-5
b_:5-7
